.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -14,8 +14,8 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        bwc_version : [ "1.1.0", "1.2.4", "1.3.7", "2.0.1", "2.1.0", "2.2.1", "2.3.0", "2.4.1" ]
-        opensearch_version : [ "2.5.0-SNAPSHOT" ]
+        bwc_version : [ "1.1.0", "1.2.4", "1.3.7", "2.0.1", "2.1.0", "2.2.1", "2.3.0", "2.4.1", "2.5.0-SNAPSHOT" ]
```

Comment:
will we need a second PR after 2.5 release to remove SNAPSHOT? I do not remember how it was done for earlier releases, did we wait for release and then upgrade or do in 2 steps

Reply:
Yes, we need to. Problem is that bwc tests are failing on main because we dont have 2.6 snapshot. So we need to upgrade 2.x to 2.6. Need to add 2.5-SNAP to bwc for sake of completeness. Once we have release we can upgrade.